GameData/SPEngine/Families/M.cfg

```diff
@@ -34,7 +34,7 @@ SPEFamily:NEEDS[RealFuels]
 		entryCost = 200000
 		entryCost = HydroloxPumps
 		entryCost = AJTitan
-		cost = 12000 // yields 7979 for an M-1-Spec
+		cost = 13850 // yields 7979 for an M-1-Spec
```

Comment:
If you've changed the cost, the comment can't still be accurate...

Reply:
Also, iirc I deliberately made the M-class cheaper than RO has it, because otherwise there is no reason in hell ever to use it — even after my buff, it's still more expensive than an equivalent cluster of Js.

Reply:
If you want me to make it cheaper then I can, but for j-2 equivilants see the J-class of engines, M-1 engines have superior performance at sea level and work more like sustainers than pure vaccum engines like the J-2's do. In a sense they're bulkier and early versions of the RS-27 engines

Reply:
Changes have been reverted